server/src/main/java/com/scalar/db/server/ServerConfig.java

```diff
@@ -1,11 +1,8 @@
-package com.scalar.db.server.config;
+package com.scalar.db.server;
```

Comment:
I moved `ServerConfig` from the `com.scalar.db.server.config` package to the `com.scalar.db.server` package.